tests/Microsoft.Identity.Test.Unit/ManagedIdentityTests/ImdsV2Tests.cs

```diff
@@ -47,15 +48,14 @@ private static readonly Func<AttestationTokenInput, CancellationToken, Task<Atte
                     AttestationToken = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.fake.attestation.sig"
                 });
 
-        public const string Bearer = "Bearer";
+        public const string Bearer = "Bearer"; 
         public const string MTLSPoP = "mtls_pop";
 
         private void AddMocksToGetEntraToken(
             MockHttpManager httpManager,
             UserAssignedIdentityId userAssignedIdentityId = UserAssignedIdentityId.None,
             string userAssignedId = null,
-            string certificateRequestCertificate = TestConstants.ValidRawCertificate,
-            bool mTLSPop = false)
+            string certificateRequestCertificate = TestConstants.ValidRawCertificate)
```

Comment:
This valid RAW certificate is expired. Can we extend this to 20 years or something?